applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -291,15 +291,15 @@ static void cell_pos_req_wk(struct k_work *work)
 	ARG_UNUSED(work);
 
 	if (cell_pos_type == CELL_POS_TYPE_SINGLE) {
-		err = nrf_cloud_cell_pos_request(NULL, true);
+		err = nrf_cloud_cell_pos_request(NULL, true, NULL);
```

Comment:
It's OK to assign NULL as of now, but I'll change to use callback for cell_pos result after merging of this PR, instead of parsing the cloud data based on run_type.